vite-dev-entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+
+yarn install
+bin/vite dev
```

Comment:
For internal observability, something like:

```suggestion
# (Add Bash to parse `--vite-env-var` argument values)

$VITE_ENV_VARS bin/vite dev bin/vite dev
```

With `$VITE_ENV_VARS` evaluated, the command could ultimately execute something like: 
> `VITE_FRONTEND_SERVICE_WORKER_CACHE="true" bin/vite dev` 

or

> `VITE_DEV_SETUP="docker-compose" VITE_FRONTEND_SERVICE_WORKER_CACHE="true" bin/vite dev`

Reply:
Sadly, `getopts` does not support long-form flags, and Mac has [well-known issues](https://stackoverflow.com/questions/402377/using-getopts-to-process-long-and-short-command-line-options) with `getopt`, which is what actually supports those.  That aside, I think this is doable.

Reply:
[0ed7c91](https://github.com/broadinstitute/single_cell_portal_core/pull/1641/commits/0ed7c91d3d2fbbf8d18aed7fdfbc1ab1b76322d7)